en/simulation/ignition_gazebo.md

```diff
@@ -56,12 +56,127 @@ In order to run the simulation without running the ignition gazebo gui, one can
+  - The setting is mutually exclusive with `PX4_GZ_MODEL`.
+
+- `PX4_GZ_MODEL`:
+  Sets the name of a new Ignition model to be spawned in the simulator.
+  If provided, the startup script looks for a model in the Ignition resource path that matches the given variable, spawns it and binds a new PX4 instance to it.
+  - The setting is mutually exclusive with `PX4_GZ_MODEL_NAME`.
+
+  :::note
+  If both `PX4_GZ_MODEL_NAME` and `PX4_GZ_MODEL` are not given, then PX4 looks for `PX4_SIM_MODEL` and uses it as an alias for `PX4_GZ_MODEL`.
+  However, this prevents the use of `PX4_GZ_MODEL_POSE`.
+  :::
+
+- `PX4_GZ_MODEL_POSE`:
+  Sets the spawning position and orientation of the model when `PX4_GZ_MODEL` is adopted.
+  If provided, the startup script spawns the model at a pose following the syntax `"x,y,z,roll,pitch,yaw"`, where the positions are given in metres and the angles are in radians.
+  - If omitted, the zero pose `[0,0,0,0,0,0]` is used.
+  - If less then 6 values are provided, the missing ones are fixed to zero.
+  - The pose requires a model; this can only be set if either of `PX4_GZ_MODEL_NAME` or `PX4_GZ_MODEL` is set.
+
+- `PX4_GZ_WORLD`:
+  Sets the Ignition world file for a new simulation.
+  If it is not given, then [default](https://github.com/PX4/PX4-Autopilot/blob/main/Tools/simulation/gz/worlds/default.sdf) is used.
+  - This variable is ignored if an existing simulation is already running.
+  - This value should be [specified for the selected airframe](#adding-new-worlds-and-models) but may be overridden using this argument.
+
+- `PX4_SIMULATOR=GZ`:
+  Sets the simulator, which for Ignition Gazebo must be `gz`.
+  - This value should be [set for the selected airframe](#adding-new-worlds-and-models), in which case it does not need to be set as an argument.
+
+The PX4 Ignition worlds and and models databases [can be found on Github here](https://github.com/PX4/PX4-Autopilot/tree/main/Tools/simulation/gz).
+They are added to the Ignition search `PATH` by [gazebo_env.sh.in](https://github.com/PX4/PX4-Autopilot/blob/main/src/modules/simulation/gz_bridge/gazebo_env.sh.in) during the simulation startup phase.
+
+:::note
+`gazebo_env.sh.in` is compiled and made available in `$PX4_DIR/build/px4_sitl/rootfs/gazebo_env.sh`
+:::
+
+### Examples
+
+Here are some examples of the different scenarios covered above.
+
+1. **Start simulator + default world + spawn vehicle at default pose**
+
+   ```sh
+   PX4_SYS_AUTOSTART=4001 PX4_GZ_MODEL=x500 ./build/px4_sitl_default/bin/px4
+   ```
+
+2. **Start simulator + default world + spawn vehicle at custom pose (y=2m)**
+
+   ```sh
+   PX4_SYS_AUTOSTART=4001 PX4_GZ_MODEL_POSE="0,2" PX4_GZ_MODEL=x500 ./build/px4_sitl_default/bin/px4
+   ```
+
+3. **Start simulator + default world + link to existing vehicle**
+
+   ```sh
+   PX4_SYS_AUTOSTART=4001 PX4_GZ_MODEL_NAME=x500 ./build/px4_sitl_default/bin/px4
+   ```
+
```

Comment:
As a post process, should we add an example with PX4_GZ_MODEL_NAME and PX4_GZ_MODEL omitted (and a note that the pose cannot be specified).

Reply:
It can be done, yes!